ReadMe.md

```diff
@@ -180,4 +184,5 @@ Your company | 御社 | おんしゃ |
 Office | 事務所 | じむしょ |
 In-house | 社内 | しゃない |
 External / 3rd party | 外部 | がいぶ |
+Contractor | 外注先 | がいちゅうさき |
```

Comment:
Double checked this with a few sources
勉強になりました、ありがとうございます
For clarification, in case anyone might read this, "契約社員" is similar to a full time employee in terms of benefits (insurances, pension etc), however their employment contract is conditional (time period etc). However "外注" is specifically someone you have contracted from the outside (outsourced) to do work for you, in other words they are not tied to any of the company benefits.
LGTM 👍